### DISCUSSION TITLE
360 ° video playback capability on mobile devices

### DISCUSSION THREAD

**Discussion by warlocks365** [immich-app/immich#13154] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Do you have any plans to add 360° video playback capability to mobile devices in the new year? At present, playing 360° videos on the mobile phone through the web page is too laggy, even under WIFI. I hope you can add the function of 360° video playback on mobile devices. It is best to have gyroscope sensing capability. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5130
